tools/kompos/index.html

```diff
@@ -400,12 +411,16 @@
   <button id="bp-menu-btn" class="btn btn-secondary btn-sm btn-block bp-btn"></button>
 
   <div id="debugger-button-icons">
-    <i id="icon-play"        class="fa fa-play"></i>
-    <i id="icon-pause"       class="fa fa-pause"></i>
-    <i id="icon-stop"        class="fa fa-stop"></i>
-    <i id="icon-arrow-down"  class="fa fa-share" style="transform: rotate(90deg);"></i>
-    <i id="icon-arrow-right" class="fa fa-share"></i>
-    <i id="icon-arrow-left"  class="fa fa-reply" style="transform: scaleY(-1);"></i>
+    <i id="icon-play"         class="fa fa-play"></i>
+    <i id="icon-pause"        class="fa fa-pause"></i>
+    <i id="icon-stop"         class="fa fa-stop"></i>
+    <i id="icon-arrow-down"   class="fa fa-share" style="transform: rotate(90deg);"></i>
+    <i id="icon-arrow-right"  class="fa fa-share"></i>
+    <i id="icon-arrow-left"   class="fa fa-reply" style="transform: scaleY(-1);"></i>
+    <i id="icon-msg-open"     class="fa fa-envelope-open"></i>
```

Comment:
which icons do you suggest? I think will be better to differentiate them somehow from the ones of sequential stepping. Right now there are a lot of arrows for the buttons I think